diff --git a/acinclude.m4 b/acinclude.m4
index 787511f..bdf0109 100644
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -609,7 +609,7 @@ GMP_PROG_CC_WORKS_PART([$1], [long long reliability test 1],
 
 #if defined (__GNUC__) && ! defined (__cplusplus)
 typedef unsigned long long t1;typedef t1*t2;
-void g(int,t1 const*,t1,t2,t1 const*,int){}
+void g(int p1,t1 const* p2,t1 p3,t2 p4,t1 const* p5,int p6){}
 void h(){}
 static __inline__ t1 e(t2 rp,t2 up,int n,t1 v0)
 {t1 c,x,r;int i;if(v0){c=1;for(i=1;i<n;i++){x=up[i];r=x+1;rp[i]=r;}}return c;}
diff --git a/configure b/configure
index 2794260..8df19f4 100755
--- a/configure
+++ b/configure
@@ -6568,7 +6568,7 @@ if test "$gmp_prog_cc_works" = yes; then
 
 #if defined (__GNUC__) && ! defined (__cplusplus)
 typedef unsigned long long t1;typedef t1*t2;
-void g(int,t1 const*,t1,t2,t1 const*,int){}
+void g(int p1,t1 const* p2,t1 p3,t2 p4,t1 const* p5,int p6){}
 void h(){}
 static __inline__ t1 e(t2 rp,t2 up,int n,t1 v0)
 {t1 c,x,r;int i;if(v0){c=1;for(i=1;i<n;i++){x=up[i];r=x+1;rp[i]=r;}}return c;}
